pkgs/tools/backup/znapzend/default.nix

```diff
@@ -54,27 +59,9 @@ stdenv.mkDerivation {
   '';
 
   postInstall = ''
-    substituteInPlace $out/bin/znapzend --replace "${perl}/bin/perl" \
-      "${perl}/bin/perl \
-      -I${Mojolicious-8-35}/${perl.libPrefix} \
-      -I${perlPackages.TAPParserSourceHandlerpgTAP}/${perl.libPrefix} \
-      -I${MojoIOLoopForkCall-0-20}/${perl.libPrefix} \
-      -I${perlPackages.IOPipely}/${perl.libPrefix} \
-      "
-    substituteInPlace $out/bin/znapzendzetup --replace "${perl}/bin/perl" \
-      "${perl}/bin/perl \
-      -I${Mojolicious-8-35}/${perl.libPrefix} \
-      -I${perlPackages.TAPParserSourceHandlerpgTAP}/${perl.libPrefix} \
-      -I${MojoIOLoopForkCall-0-20}/${perl.libPrefix} \
-      -I${perlPackages.IOPipely}/${perl.libPrefix} \
-      "
-    substituteInPlace $out/bin/znapzendztatz --replace "${perl}/bin/perl" \
-      "${perl}/bin/perl \
-      -I${Mojolicious-8-35}/${perl.libPrefix} \
-      -I${perlPackages.TAPParserSourceHandlerpgTAP}/${perl.libPrefix} \
-      -I${MojoIOLoopForkCall-0-20}/${perl.libPrefix} \
-      -I${perlPackages.IOPipely}/${perl.libPrefix} \
-      "
+    substituteInPlace $out/bin/znapzend      --replace "${perl}/bin/perl" "${perl'}/bin/perl"
+    substituteInPlace $out/bin/znapzendzetup --replace "${perl}/bin/perl" "${perl'}/bin/perl"
+    substituteInPlace $out/bin/znapzendztatz --replace "${perl}/bin/perl" "${perl'}/bin/perl"
```

Comment:
I have not checked, but does `substituteInPlace` do anything here?
Since we have `perl'` instead of `perl` in `buildInputs = []`, how `${perl}/bin/perl` could appear inside the files?
`patchShebang .` is to put `${perl'}/bin/perl` there, so `substituteInPlace` seems unnecessary

Reply:
Right, it's actually useless: I have just checked.